### PR TITLE
set buildName on dynamic GradleBuild when using gradle 6.x or higher

### PR DESCRIPTION
**Pull request by tomnis**
- see #27 for more detail

### CODE REVIEW

File: src/main/groovy/com/adtran/ScalaMultiVersionPlugin.groovy [hunk 1]
```diff
@@ -130,6 +132,9 @@ class ScalaMultiVersionPlugin implements Plugin<Project> {
                         startParameter.projectProperties["recursed"] = true
                         startParameter.excludedTaskNames += getRunOnceTasks()
                         tasks = project.gradle.startParameter.taskNames
+                        if (6 <= project.gradle.gradleVersion.split(Pattern.quote('.'))[0].toInteger() ){
```
Comment (nathanalderson) [inline src/main/groovy/com/adtran/ScalaMultiVersionPlugin.groovy:137]: I think this deserves a comment, please. At least a reference to the issue number.

**Review by nathanalderson — CHANGES_REQUESTED** [review]

File: src/main/groovy/com/adtran/ScalaMultiVersionPlugin.groovy (hunk 1 above)
Comment (nathanalderson) [reply src/main/groovy/com/adtran/ScalaMultiVersionPlugin.groovy:137]:
```suggestion
                        // workaround name clash issue in Gradle 6+. See:
                        // https://github.com/ADTRAN/gradle-scala-multiversion-plugin/issues/27.
                        if (6 <= project.gradle.gradleVersion.split(Pattern.quote('.'))[0].toInteger() ){
```

Comment (tomnis) [reply src/main/groovy/com/adtran/ScalaMultiVersionPlugin.groovy:137]: cool, good call

**Review by nathanalderson — APPROVED** [review]

**Comment by nathanalderson** [comment]
Despite the fact that the last two builds failed, it seems like it actually did publish. Not sure what's going on there. Let me know if you're not able to pull the updated version.

**Comment by tomnis** [comment]
I can confirm we are able to pull 1.0.36. thanks!

**Pull request merged**